### PR TITLE
Move matrix config to info.json, part 5

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

#20001 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/woodkeys/scarletbandana/info.json [hunk 1]
```diff
@@ -8,6 +8,11 @@
         "pid": "0x6969",
         "device_version": "0.0.1"
     },
+    "matrix_pins": {
+        "cols": ["D3", "D5", "D4", "D6", "D7", "B4", "B5", "B6", "C6", "C7", "F7", "F6", "E6"],
+        "rows": ["F0", "F1", "F4", "F5"]
```
Comment (tzarc) [inline keyboards/woodkeys/scarletbandana/info.json:13]:
```suggestion
        "cols": ["B2", "B3", "B7", "B1", "F5", "F4", "F6", "F7", "B0", "F0", "F1", "D0", "D1", "D2", "D5", "B6", "C7"],
        "rows": ["D4", "D6", "D7", "B4", "B5"]
```

**Review by tzarc — CHANGES_REQUESTED** [review]

File: keyboards/skme/zeno/info.json [hunk 2]
```diff
@@ -8,6 +8,11 @@
         "pid": "0x0001",
         "device_version": "0.0.1"
     },
+    "matrix_pins": {
```
Comment (tzarc) [inline keyboards/skme/zeno/info.json:11]: After merging, this causes issues -- seems like this file has already been migrated on `develop`?

See https://github.com/qmk/qmk_firmware/blob/develop/keyboards/skme/zeno/info.json.

**Review by tzarc — CHANGES_REQUESTED** [review]
Two identified problems, one of them probably because it was migrated elsewhere.

Also, for some reason `studiokestra/galatea/rev1:default` fails to build after merge, too:
```
Linking: .build/studiokestra_galatea_rev1_default.elf                                               [ERRORS]
 |
 | /usr/lib/gcc/avr/12.1.0/../../../../avr/bin/ld: .build/obj_studiokestra_galatea_rev1_default/quantum/action.o: in function `process_action':
 | /home/nickb/qmk_build/sha1_verification/qmk_firmware_pr/quantum/action.c:585: undefined reference to `layer_state_set'
 | collect2: error: ld returned 1 exit status
 |
make[1]: *** [builddefs/common_rules.mk:267: .build/studiokestra_galatea_rev1_default.elf] Error 1
Make finished with errors
```

File: keyboards/skme/zeno/info.json (hunk 2 above)
Comment (tzarc) [reply keyboards/skme/zeno/info.json:11]: Looks like #20023.

File: keyboards/skme/zeno/info.json [hunk 3]
```diff
@@ -8,6 +8,11 @@
...
+        "cols": ["F0", "F1", "F4", "F5", "F6", "F7", "B0", "C6", "B6", "B5", "D5", "D3", "D2", "D1", "D0"],
+        "rows": ["B1", "B2", "B3", "B7", "C7"]
+    },
+    "diode_direction": "COL2ROW",
```
Comment (fauxpark) [inline keyboards/skme/zeno/info.json:15]:
```suggestion
```
Let's undo this then.

**Review by tzarc — APPROVED** [review]

**Pull request merged**